.circleci/windows/csharp/deploy_release.bat

```diff
@@ -0,0 +1,29 @@
+@echo off
+REM Licensed to the Apache Software Foundation (ASF) under one
+REM or more contributor license agreements.  See the NOTICE file
+REM distributed with this work for additional information
+REM regarding copyright ownership.  The ASF licenses this file
+REM to you under the Apache License, Version 2.0 (the
+REM "License"); you may not use this file except in compliance
+REM with the License.  You may obtain a copy of the License at
+REM
+REM   http://www.apache.org/licenses/LICENSE-2.0
+REM
+REM Unless required by applicable law or agreed to in writing,
+REM software distributed under the License is distributed on an
+REM "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+REM KIND, either express or implied.  See the License for the
+REM specific language governing permissions and limitations
+REM under the License.
+
+REM needs to be called such that software installed
+REM by Chocolatey in prepare.bat is accessible
+CALL refreshenv
+
+ECHO Building and deploying windows package...
+SET DEPLOY_NUGET_API_KEY=%REPO_TYPEDB_PASSWORD%
+@REM TODO: Add new key for nuget as a new variable!
```

Comment:
Only for release, we'll configure it with @flyingsilverfin once the nuget bot account with the key is created.